pkg/stub/handler.go

```diff
@@ -28,7 +28,12 @@ type Handler struct {
 	manifestFactory *manifests.Factory
 }
 
+var ci_md5 map[string][16]byte
+
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
+	if ci_md5 == nil {
+		ci_md5 = make(map[string][16]byte)
```

Comment:
Where is this used? probably stale code

Reply:
Difference in MD5 checksum of the object to denote change in custom resource may not be useful in our case.
Once we populate status field, status is updated periodically and md5 checksum on the object will change even though there is no change in CR spec.
I think we should store old spec (as annotation or something mechanism) and then compare with new spec to determine if update is needed. This mechanism will also provide what exactly has changed and can react accordingly.
We could get more pointers from `oc apply` implementation.